src/go/rdctl/pkg/utils/utils_windows.go

```diff
@@ -42,17 +42,16 @@ func GetRDPath() (string, error) {
 	dir, err := directories.GetLocalAppDataDirectory()
 	if err == nil {
 		dataPaths = append(dataPaths, dir)
+	} else {
+		dataPaths = append(dataPaths, filepath.Join(homedir, "AppData", "Local"))
 	}
 	// %APPDATA%
 	dir, err = directories.GetRoamingAppDataDirectory()
 	if err == nil {
 		dataPaths = append(dataPaths, dir)
+	} else {
+		dataPaths = append(dataPaths, filepath.Join(homedir, "AppData", "Roaming"))
 	}
```

Comment:
I think this block can be deleted completely. We should **never** install into the Roaming directory.

Reply:
And the part below about lines 45-46 of `utils_windows.go`, falling back if `GetLocalAppDataDirectory()` fails: it's gone